### DISCUSSION TITLE
ClickBench results of CrateDB 2 years out of date

### DISCUSSION THREAD

**Discussion by robd003** [crate/crate#16025] [General]
Just wondering if it's worth trying to get the results updated with the current release of CrateDB. It looks like the last results.json run was from over 2 years ago.

https://github.com/ClickHouse/ClickBench/tree/main/cratedb/results

This was probably run using CrateDB 4.8.x and there's been a lot of bug fixes since then.

**Comment by amotl** [comment]
Hi. Thanks for your report. Is this something you could be looking into, @henrikingo?
